### ISSUE TITLE
Package is bundled incorrectly causing issues with types / ESM.

### ISSUE THREAD

**Issue by mattf96s** [jpwallace22/use-react-workers#5]
The package is bundled incorrectly which causes an ESM error with the default import or type errors when importing with `import { useWorkerFunc } from 'use-react-workers/dist/index.mjs'`.

**To Reproduce**
Visit [arethetypeswrong](https://arethetypeswrong.github.io/?p=use-react-workers%400.3.0).

**Expected behavior**
The package is usable within an ESM project without type errors.

**Comment by jpwallace22** [comment]
@mattf96s Thanks for the issue! Couple questions. Why are you importing it from `dist/index.mjs` like that? If you were to import:
```js
import { useWorkerFunc } from 'use-react-workers`
```
You would be getting the correct types. 

The `aretypeswrong` indication is that the type declarations don’t represent the runtime JavaScript. This is because the types are generated via a rollup plugin under certain settings, while the JavaScript is generated by another tool entirely, this IS something that can be/will changed but it still wouldn't resolve the issue if you're importing directing from the `index.mjs`.

**Comment by jpwallace22** [comment]
Issue was autoclosed with merging of possible fix. Reopening until everything can be confirmed.

**Comment by rafark** [comment]
> 🎉 This issue has been resolved in version 1.0.0 🎉
> 
>

No it has not:

```
TS2305: Module '"use-react-workers"' has no exported member 'useWorkerFunc'.
    1 | import { Editor, Element, Frame, useEditor, useEditorReturnType, useNode } from "@craftjs/core";
    2 | import { Children, FC, Fragment, ReactNode, isValidElement, useEffect } from "react";
    3 | import {useWorkerFunc} from 'use-react-workers';
```

**Comment by jpwallace22** [comment]
@rafark This comment is less than helpful. Are you able to provide a bit more info about your situation/project? 

According to the [arethetypeswrong](https://arethetypeswrong.github.io/?p=use-react-workers%401.0.0) link (as well as my testing) they are passing.
